app/templates/partials/calendar/monthly_view/add_week.html

```diff
@@ -15,6 +15,15 @@
         {% for event in day.events %}
           <div class="{{ day.css['event'] }}"><b>{{ event[0] }}</b> {{ event[1] }}</div>
         {% endfor %}
+        {% for parasha in parashot %}
+            {% if parasha.date.date() == day.date %}
+            <div class="parashot">
+                <p>{{parasha.name}} / {{parasha.hebrew_name}}<br>
+                    <a style="background-color:powderblue;" href={{parasha.link}} target="_blank">
+                        <b>פירוט בלינק נפרד על {{parasha.hebrew_name}}</b></a></p>
```

Comment:
you could use gettext here

Reply:
What PD said. Make sure the default text is in English as that is our base language.
Then do something like:

`{{ gettext("Parashah on %(parashah)s", parashah=parasha.hebrew_name) }}`

* {{ }} - start of code.
* gettext() - tells the system that the text is translatable.
* "" - the text itself; in English.
* %()s - a named parameter. Must start with "%" and end with "s".
* parashah= - the named parameter. Must match the named inside the "%()s".
* parasha.hebrew_name - the value.

Hope this helps.

Reply:
and if my text in hebrew?

Reply:
I EDIT.THANKS